doc/source/methods/Anchors.ipynb

```diff
@@ -64,7 +64,9 @@
     "\n",
```

Comment:
<p>can be sampled in 2 ways --&gt; can be sampled in 3 ways</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/432/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
changed